src/components/attendee/Card/CardButton.tsx

```diff
@@ -0,0 +1,9 @@
+import { Button, ButtonProps } from "components/attendee/Button";
+
+export const CardButton: React.FC<ButtonProps> = ({ children, ...rest }) => {
+  return (
+    <Button {...rest} borderRadius="none" margin="none">
```

Comment:
[NOTE]
I cant say if and how it will reflect this case, might be the above line is quite OK, so I will just note that generally, for DX reasons:

- margins and basically all of layout would be the concern of the parent, not the components inside. Flexbox for example can use `padding`, `gap` etc.

- some times props that work like booleans can be named `marginless` or `rounded` depending on the case itself

Reply:
added `marginless` or `rounded` properties:
https://github.com/sparkletown/sparkle/pull/2953/commits/a52527d7fce74b5f15447723260cc7715c8614b3